sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md

```diff
@@ -0,0 +1,22 @@
+---
+title: AzureResourceManager
+key: azureresourcemanager
+---
+
+## Supported Versions
+
+| Format | Version    | Status              |
+|--------|------------|---------------------|
+| JSON   | 2023-06-01 | Partially Supported |
```

Comment:
It is not clear to me what version of JSON format should we put here. I choose the newest stable version from here: https://github.com/Azure/azure-resource-manager-schemas/tree/main/schemas Feel free to challenge it.

Reply:
maybe we can say `up to` since we are not aware of any problems.